base/opaque_closure.jl

```diff
@@ -0,0 +1,23 @@
+function show(io::IO, oc::Core.OpaqueClosure{A, R}) where {A, R}
+    show_tuple_as_call(io, Symbol(""), A; hasfirst=false)
+    print(io, "::", R)
+    print(io, "->◌")
+end
+
+function show(io::IO, ::MIME"text/plain", oc::Core.OpaqueClosure{A, R}) where {A, R}
+    show(io, oc)
```

Comment:
Let's avoid specialization
```suggestion
function show(io::IO, ::MIME"text/plain", oc::Core.OpaqueClosure)
    show(io, oc)
```